README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
+	DT = [netcdftime.datetime(year=2017, month=2, day=day) for day in range(1, 31)]
+	CDT = [nc_time_axis.CalendarDateTime(item, "360_day") for item in DT]
+	TempList = []
```

Comment:
In scitools we tend to name variables in lowercase with underscores. You will also find matplotlib, numpy and similar libraries do this too.
It's also best to have descriptive names so that your code is more readable. 

Could you rename the above as something along the lines of
`DT` -> `dt` or `datetimes` or `dates` etc
`CDT` -> `cdt` or `calendar_datetimes` or `cal_datetimes` etc
`TempList` -> `temperature(s)`